pkg/gardenlet/controller/seed/seed/components.go

```diff
@@ -70,7 +70,9 @@ func defaultKubeStateMetrics(
 		ClusterType: component.ClusterTypeSeed,
 		Image:       image.String(),
 		Replicas:    1,
-	}), nil
+	},
+		false,
```

Comment:
What is the impact of defaulting `isWorkerless` to false?

Reply:
isWorkerless is only relevant for shoots. For seed, setting it to false doesn't affect anything because nothing depends on this field in seed `kube-state-metrics` deployment.